tools/deploy_prow.sh

```diff
@@ -47,6 +47,12 @@ function launchProwConfig() {
   # PR Status
   kubectl create secret generic github-oauth-config --from-literal=secret="$(./tools/1password.sh -d prow-prstatus-github-oauth-app.yaml)" || true
   kubectl create secret generic cookie --from-literal=secret="$(./tools/1password.sh -d prow-prstatus-cookie-encryption-key.txt)" || true
+
```

Comment:
Despite `cosign` is used with this PR with Kubernetes Secrets as backend, we keep 1Password as the source of truth for secrets (still, we need to better automate or change system).

Reply:
cc @zuc